### ISSUE TITLE
Search doesn't work

### ISSUE THREAD

**Issue by hgabe** [FaisalUmair/udemy-downloader-gui#60]
The search bar doesn't work well in searching for specific courses

**Comment by FaisalUmair** [comment]
Yes, there are quite a few issues regarding the search here. It is just a filter which filters the courses based on their names. It only works with the courses loaded in the UI. 

In next release, I will add a proper search, which will search the entire user subscribed courses list.

**Comment by hgabe** [comment]
thanks
